src/components/YourCompany/BusinessContactInfo.vue

```diff
@@ -1,293 +1,77 @@
-          </v-flex>
-          <v-flex xs10>
-            <v-text-field
-              filled
-              label="Email Address"
-              req
-              persistent-hint
-              :rules="emailRules"
-              v-model="contact.email"
-              id="txt-email"
-            >
-            </v-text-field>
-          </v-flex>
-        </v-layout>
-
-        <!-- Line 2 -->
-        <v-layout row class="mx-0">
-          <v-flex xs2>
-            <label><strong>Confirm Email</strong></label>
-          </v-flex>
-          <v-flex xs10>
-            <v-text-field
-              filled
-              label="Confirm Email Address"
-              req
-              persistent-hint
-              :error-messages="emailMustMatch()"
-              v-model="contact.confirmEmail"
-              id="txt-confirm-email"
-            >
-            </v-text-field>
-          </v-flex>
-        </v-layout>
-
-        <!-- Line 3 -->
-        <v-layout row class="mx-0">
-          <v-flex xs2>
-            <label><strong>Phone Number</strong></label>
-          </v-flex>
-          <v-flex xs6 class="pr-3">
-            <v-text-field
-              filled
-              label="Phone Number"
-              persistent-hint
-              hint="Example: (555) 555-5555"
-              type="tel"
-              v-mask="['(###) ###-####']"
-              v-model="contact.phone"
-              :rules="phoneRules"
-              id="txt-phone"
-            >
-            </v-text-field>
-          </v-flex>
-          <v-flex xs4>
-            <v-text-field
-              filled
-              label="Extension"
-              persistent-hint
-              v-mask="'#####'"
-              v-model="contact.extension"
-              :disabled="!contact.phone"
-              id="txt-phone-extension"
-            >
-            </v-text-field>
-          </v-flex>
-        </v-layout>
-
-        <!-- Buttons -->
-        <v-layout row class="mx-0 pt-3">
-          <v-flex xs12>
-            <div class="action-btns">
-              <v-btn
-                id="done-btn"
-                large color="primary"
-                :disabled="!formValid"
-                @click="updateContactInfo()"
-              >
-                <span>Done</span>
-              </v-btn>
-              <v-btn
-                id="cancel-btn"
-                large outlined color="primary"
-                @click="resetContactInfo()"
-              >
-                <span>Cancel</span>
-              </v-btn>
-            </div>
-          </v-flex>
-        </v-layout>
-      </v-form>
-    </template>
-  </div>
+  <contact-info
+    :businessContact="getBusinessContact"
+    :originalBusinessContact="originalContact"
+    :hasBusinessContactInfoChange="hasBusinessContactInfoChange"
+    :edit-label="editLabel"
+    :edited-label="editedLabel"
+    @contactInfoChange="setContact($event)"
+    @haveChanges="emitHaveChanges($event)"
+  />
```

Comment:
I noticed, while testing Contact Info in Storybook, that anew email address isn't "saved" to the read-only version. Does it work here?

Reply:
Good point, the values displayed in the read only version are straight from store.
So in storybook changes made aren't reflected there since those values are not stored local to the component, although there could be a way to better mock this in storybook.

It does work when we are able to update the values we pass into the Shared Component, you bet.

Reply:
Maybe the Storybook component needs a handler for `@contactInfoChange`.
(Maybe fix it next time we're in there?)

Reply:
Maybe we could get the change notification working in SB also :)

Reply:
Yes and yes, absolutely will do following this PR.

I made some changes so that the read only portion of the component wasn't local data and this was a byproduct of that, i didn't notice until now!